.github/workflows/regression-testing/regressions-util.py

```diff
@@ -0,0 +1,494 @@
+from typing import Tuple
+
+import os
+import sys
+import argparse
+import numpy as np
+import pandas as pd
+
+CLASSIFICATION_METRICS = ['acc', 'auc', 'balacc', 'logloss']
+REGRESSION_METRICS = ['mae', 'r2', 'rmse']
+METRICS = CLASSIFICATION_METRICS + REGRESSION_METRICS
+TARGET_YAML_FILE = 'target.yaml'
+
+def _get_mean_results_across_folds(df) -> pd.DataFrame:
```

Comment:
This function is identical to the function in the other python file.

Reply:
The regression-testing folder wasn't meant to be included, I'll delete this